### DISCUSSION TITLE
What do you think about field comments?

### DISCUSSION THREAD

**Discussion by SebastiaanKloos** [filamentphp/filament#1506] [Ideas]
Sometimes you want to provide some additional usage information for fields. I've seen some applications placing a comment just below the label or the field. 

What do you think about a feature for all fields like this?:

```php
// For comments just after the label
TextInput::make('name')
    ->comment('Fill in the name of the person'),

// Or for placing comments below the field itself
TextInput::make('name')
    ->commentBelow('Fill in the name of the person'),
```

I'm more than happy to create a PR if accepted :-)

**Comment by danharrin** [comment]
We have `->hint()` (for after the label) and `->helperText()` (for below the label) :)

**Reply by SebastiaanKloos** [reply]
@danharrin ahh didn't know that! Many thanks!
